wherehows-data-model/DDL/ETL_DDL/etl_configure_tables.sql

```diff
@@ -13,82 +13,42 @@
 --
 
 -- configuration tables
-CREATE TABLE `wh_etl_job` (
-  `wh_etl_job_id`   INT(10) UNSIGNED  NOT NULL AUTO_INCREMENT
-  COMMENT 'id of the etl job',
-  `wh_etl_job_name` VARCHAR(127)      NOT NULL
+CREATE TABLE `wh_etl_job_schedule` (
+  `wh_etl_job_name` VARCHAR(127)  NOT NULL
   COMMENT 'etl job name',
-  `wh_etl_type`     VARCHAR(127)               DEFAULT NULL
-  COMMENT 'type of the etl service',
-  `cron_expr`       VARCHAR(127)               DEFAULT NULL
-  COMMENT 'frequency in crob expression',
-  `ref_id`          SMALLINT(6)       NOT NULL
-  COMMENT 'id of application/database where etl job runs',
-  `ref_id_type`     ENUM('APP', 'DB') NOT NULL
-  COMMENT 'flag the ref_id is an application or a database',
-  `timeout`         INT(11)                    DEFAULT NULL
-  COMMENT 'timeout in seconds for this etl job',
-  `next_run`        INT(10) UNSIGNED           DEFAULT NULL
+  `next_run` INT(10) UNSIGNED     DEFAULT NULL
```

Comment:
In the future if we move on to Gobblin based ETL, then this table can also be deprecated.